src/odemis/driver/hamamatsurx.py

```diff
@@ -676,6 +680,18 @@ def _setMCPGain(self, value):
 
         return value
 
+    def _getStreakUnitMCPGainRange(self):
+        """
+        Get range for streak unit MCP gain.
+        First 5 values see CamParamInfoEx.
```

Comment:
Move this "First 5 values see CamParamInfoEx" to a comment inside the function. It doesn't help to understand how to call that function.